@xen-orchestra/lite/src/views/pool/PoolVmsView.vue

```diff
@@ -1,6 +1,12 @@
 <template>
   <UiCard class="pool-vms-view">
-    <VmsActionsBar :selected-refs="selectedVmsRefs" />
+    <UiCardTitle>
+      VMs
```

Comment:
Following your message, when I started working on the i18n of this file I realized that other translations were missing.

Little by little I found myself internationalizing the filtering (which I wanted to do for a long time, but which required a big rewrite of the code because of the too complicated selects to translate).

Therefore, it would probably be best to merge this PR and make another one for the i18n.